hack/build-appregistry.sh

```diff
@@ -3,8 +3,9 @@
 set -e
 
 source hack/common.sh
+source hack/containers-common.sh
 source hack/generate-appregistry.sh
 
 $IMAGE_BUILD_CMD build --no-cache -t "$CATALOG_FULL_IMAGE_NAME" -f build/Dockerfile.registry .
 echo
-echo "Run '${IMAGE_BUILD_CMD} push ${CATALOG_FULL_IMAGE_NAME}' to push ocs-registry to ${IMAGE_REGISTRY}."
\ No newline at end of file
+echo "Run '${IMAGE_BUILD_CMD} push ${CATALOG_FULL_IMAGE_NAME}' to push ocs-registry to ${IMAGE_REGISTRY}."
```

Comment:
I still see this change... is this your editor doing this? I would either remove this change or move it into its own commit, since it is an unrelated formatting change.

Reply:
Yes, my editor did it. Moved to separate commit